### ISSUE TITLE
Unnecessary dependency on a full Xcode installation

### ISSUE THREAD

**Issue by oxyc** [crisidev/homebrew-chunkwm#8]
Chunkwm only depends on Xcode Command Line tools to install but this formula adds the full Xcode.app as a dependency.

From the README

> Requires xcode-8 command line tools.

Error message when attempting to install without Xcode.app through this formula

> chunkwm: A full installation of Xcode.app is required to compile this software.\nInstalling just the Command Line Tools is not sufficient.
Xcode can be installed from the App Store.
Error: An unsatisfied requirement failed this build.

By removing the dependency it still builds successfully with all plugins:

```
# Chunkwm build with:
  * tiling: true
  * border: true
  * ffm: true
  * transparency: true
```

**Comment by crisidev** [comment]
Is there any homebrew reference to depend only on xcode command line tools?

Otherwise it will fail badly without it..

**Comment by oxyc** [comment]
Not sure if you can specify the version but homebrew itself depends on command line tools at least.

**Comment by crisidev** [comment]
@koekeishiya, thoughts about removing xcode dependency?

**Comment by asmvik** [comment]
Uh, I personally like to depend on as little as possible.

I have to admit I do have XCode installed for its debugger, so I cannot actually guarantee that what I stated in the build instructions of *chunkwm* applies, although it should.

**Comment by oxyc** [comment]
I'm only using the tiling plugin but that has been working for me since the pre-alpha of chunkwm. At least the builds succeed for all other plugins as well but I haven't tested their functionality.

Edit: _Command Line Tools_ are already [listed as a requirement](http://docs.brew.sh/Installation.html) in the Homebrew installation requirements.

**Comment by crisidev** [comment]
Perfect. I'll update ASAP. Thanks for spotting this.

**Comment by crisidev** [comment]
Resolved with commit 37e4c44bf26bef44a253d28e99cdcaf73048bd4f
